Clients/GUI/Shared/Currency/CurrencyContainer.razor

```diff
@@ -0,0 +1,43 @@
+﻿@using System.Net.Http
+
+@if (Currencies != null)
+{
+    <p class="badge-dark" style="margin-bottom:1px">CURRENCY RATE</p>
+    @if (CurrencyList == null)
+    {
+        @foreach (var cur in Currencies)
+        {
+            <div class="text-center container-fluid" style="white-space:nowrap">
+                <span class="item i-body">@cur.Id</span>
+                <span class="item i-body">@cur.Value</span>
+            </div>
+        }
+    }
+    else
+    {
+        @foreach (var cur in Currencies)
+        {
+            @if (CurrencyList.Contains(cur.Id))
+            {
+                <div class="text-center container-fluid" style="white-space:nowrap">
+                    <span class="item i-body">@cur.Id</span>
+                    <span class="item i-body">@cur.Value</span>
+                </div>
+            }
+        }
+    }
+}
+
+@code {
+    [Parameter]
+    public List<string> CurrencyList { get; set; }
+
+    Currency[] Currencies { get; set; }
+
+    protected override async Task OnInitializedAsync()
+    {
+        string uri = "https://localhost:44384/NewsService/currencies?newsPublisherType=CentralBank";
+        HttpClient client = new HttpClient();        
```

Comment:
Why do we use HttpClient?

Reply:
because it faster

Reply:
I heard that Spartak said something about it isn't good. Also , now we have another class that workes with httprequests. Why don't we use it?

Reply:
if you have already written such a method, then I will rewrite it. But it still has to return an object not json/string

Reply:
You could say it in reviewing and it would be easier to fix :) 
What do you mean? Just deserialize object that returned response?